src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-list/pool-list.component.html

```diff
@@ -40,7 +40,7 @@
     <ng-template ngbNavContent>
       <cd-grafana [grafanaPath]="'ceph-pools-overview?'"
                   uid="z99hzWtmk"
-                  grafanaStyle="two">
+                  grafanaStyle="three">
```

Comment:
I think `two` is enough, what might need fixing is the grafana panel itself.
```suggestion
                  grafanaStyle="two">
```